templates/kubernetes/charmed-k8s.html

```diff
@@ -6,248 +6,263 @@
+      </p>
+      <p>
+        Choose your infrastructure and seamlessly integrate with <a href="/blog/what-is-cloud-native">cloud native</a> storage, networking, load balancing and observability.
+      </p>
+    {%- endif -%}
+    {%- if slot == "cta" -%}
+      <a class="p-button--positive js-invoke-modal"
+         href="/kubernetes/contact-us?product=kubernetes-features">Get in touch</a>
+    {%- endif -%}
+    {%- if slot == "image" -%}
+      <div class="u-embedded-media">
+        <iframe width="560"
+                height="315"
+                src="https://www.youtube.com/embed/oGk8C4PDMxw?si=GNnO8_rnMmYtINJo"
+                title="Fully automated, extensible Kubernetes for all"
+                frameborder="0"
+                allow="accelerometer; autoplay;clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+                referrerpolicy="strict-origin-when-cross-origin"
+                allowfullscreen
+                class="u-embedded-media__element"></iframe>
       </div>
-    </div>
-  </section>
+    {%- endif -%}
+  {% endcall -%}
 
-  <section class="p-strip--light">
-    <div class="row">
-      <div class="col-8">
+  <section class="p-section">
+    <div class="u-fixed-width">
+      <hr class="p-rule" />
+      <div class="p-section--shallow">
         <h2>Highly configurable, pure upstream, production-grade K8s clusters</h2>
       </div>
     </div>
-    <div class="row">
-      <div class="col-12">
-        <h3 class="p-heading--4 u-sv3">CNCF-certified distribution and best-of-breed tools for a turnkey Kubernetes</h3>
-        <div class="u-hide--small u-sv3">
-          {{ image(url="https://assets.ubuntu.com/v1/f2505124-164488352-ceaad6dd-4eee-407c-9e73-64afa353226c.png",
+    <div class="p-section--shallow">
+      <div class="row">
+        <div class="col-start-large-7 col-6">
+          <h3 class="p-heading--5">CNCF-certified distribution and best-of-breed tools for a turnkey Kubernetes</h3>
+          <p>
+            Charmed Kubernetes provides automatic updates and security fixes for all core Kubernetes components and a high availability setup by default.
+          </p>
+          <p>
+            Additionally it brings full OCI compatibility with Docker and containerd runtimes as well as PCI device passthrough for GPU, FPGA and SR-IOV workloads.
+          </p>
+          <p>
+            It comes with an extensible, third-party ecosystem for storage and networking and enables VMs on Kubernetes with Kata containers.
+          </p>
+        </div>
+      </div>
+    </div>
+    <div class="u-fixed-width">
+      <div class="p-section--shallow">
+        <div class="p-image-container--3-2 is-cover is-highlighted">
+          {{ image(url="https://assets.ubuntu.com/v1/f4a59bfe-k8s-chart-updated.png",
                     alt="",
```

Comment:
For this and the chart further is was decided that these are decorative since the context is in the surrounding copy, so the missing alt text is intentional